source/features/deep-reblame.tsx

```diff
@@ -47,7 +47,7 @@ const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number):
 
 async function redirectToBlameCommit(event: DelegateEvent<MouseEvent, HTMLAnchorElement>): Promise<void> {
 	const blameLink = event.delegateTarget;
```

Comment:
This is no longer just an `<a>`

```suggestion
async function redirectToBlameCommit(event: DelegateEvent<MouseEvent, HTMLAnchorElement | HTMLButtonElement>): Promise<void> {
	const blameElement = event.delegateTarget;
```